### DISCUSSION TITLE
Cannot add Azure DevOps installation

### DISCUSSION THREAD

**Discussion by thoemmi** [getsentry/support#30] [Support]
I want to add Azure DevOps integration to our Sentry organization. However, after I select the DevOps account in the popup, it closes and the message "_The provided Azure DevOps account is linked to a different Sentry user. To continue linking the current Sentry user, please use a different Azure DevOps account._" appears at the bottom.

However, I don't know which Sentry user is linked to the DevOps account. Is there any way to find out the user in question, and to link the DevOps account to another user?

**Comment by Kobby-Bawuah** [comment]
Hello @thoemmi will it be possible to write to support about this? It will be difficult to debug without information about the issue.
